src/js/index.js

```diff
@@ -0,0 +1,101 @@
+const calculator = document.querySelectorAll(".calculator");
+const total = document.getElementById("total");
+
+// Properties
+let calculatingValue = {
+    currentValue: 0,
+    previousValue: 0,
+    operator: ""
+  };
+
+// Initial Setting
+calculator.forEach((button) => {
+    button.addEventListener("click", handleClickBtn);
+});
+
+// Functions
+function handleClickBtn(event) {
+    const value = event.target.innerText;
+
+    if (/[0-9]/.test(value)) {
+      return setNumber(value);
+    }
+  
+    if (value === "AC") {
+      return reset();
+    }
+  
+    if (value === "=") {
+      return complete();
+    }
+  
+    return setOperation(value);
+  }
+
+  function setNumber(value) {
+    let inputNumber = String(total.innerText) + value;      
+    renderNumber(String(Number(inputNumber)));
+  }
```

Comment:
```suggestion
  function setNumber(value) {
    const inputNumber = total.innerText + value;      
    renderNumber(inputNumber);
  }
```

아래 두 줄을 실행시켜보시면 모두 string이라고 나올 것에요
```
console.log(typeof ('' + 1))
console.log(typeof (1 + ''))
```

관련 아티클 일독 추천드립니다 https://poiemaweb.com/js-type-coercion